css/mark-novicio.css

```diff
@@ -0,0 +1,26 @@
+* {
+    font-family: 'Raleway', sans-serif;
+    line-height: 200%;
+}
+
+h1 {
+    color: #ff0000;
+}
+
+#first-list {
+    color: #00ff00
+}
+
+.date {
+    font-weight: bold;
+    font-style: italic;
+}
+
+li>blockquote {
+    text-decoration: underline;
+
+}
+
+li blockquote {
```

Comment:
This rule isn't being applied to the last `<blockquote>` element. Need to revise.